### DISCUSSION TITLE
Modelling sample holders

### DISCUSSION THREAD

**Discussion by aalmpani** [gimli-org/pyGIMLi#317] [Ideas]
Hello, I was wondering if there's an option for doing modelling of cylindrical sample holders with 2 current and 2 potential electrodes. Simple case scenarios, such as SIP signatures for specific homogeneous distribution, sensitivity of the array.

Thanks,
Angelos

**Comment by halbmy** [comment]
Of course that's possible. You might have a look at the BERT repository
https://gitlab.com/resistivity-net/bert
where there are several examples on that. Mostly still with bash but they can be transferred to Python easily.

**Reply by aalmpani** [reply]
Awesome, thank you!
